src/ceph-daemon/ceph-daemon

```diff
@@ -1936,12 +1939,16 @@ if __name__ == "__main__":
             except Exception as e:
                 logger.debug('could not locate %s: %s' % (i, e))
         if not container_path:
-            raise RuntimeError('unable to locate any of %s' % CONTAINER_PREFERENCE)
+            raise CDError('unable to locate any of %s' % CONTAINER_PREFERENCE)
```

Comment:
This is outside the exception handling and will produce a traceback on the console.